azure-pipelines.yml

```diff
@@ -57,15 +57,6 @@ stages:
           dotnet --version
         displayName: 'List dotnet versions and current version'
 
-      - task: Cache@2
-        displayName: Cache NuGet packages
-        inputs:
-          key: 'nuget | "$(Agent.OS)" | **/packages.lock.json'
-          restoreKeys: |
-            nuget | "$(Agent.OS)"
-            nuget
-          path: $(NUGET_PACKAGES)
-
```

Comment:
I've removed caching since this needs **RestoreLockedMode** and this does not work when targeting both .Net 4.6.1 and .Net Standard

Reply:
do you think it's worth letting the azure pipelines team know of this?

Reply:
I think nuget is the culprit here since I can reproduce this also locally. Indeed it would be nice to report an issue.